com.ryonwhyte.SimplyFrpGui.yml

```diff
@@ -0,0 +1,91 @@
+app-id: com.ryonwhyte.SimplyFrpGui
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: simply-frp-gui
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-config/simply-frp-gui:create
```

Comment:
why?

Reply:
stop using a chatbot to answer questions. Because it doesn't answer them.

The subject is line 21, and you haven't explained why.